t/ui/10-tests_overview.t

```diff
@@ -165,6 +165,7 @@ like($driver->find_elements('.failedmodule', 'css')->[1]->get_attribute('href'),
 my @descriptions = $driver->find_elements('td.name a', 'css');
 is(scalar @descriptions, 2, 'only test suites with description content are shown as links');
 $descriptions[0]->click();
+wait_for_ajax(msg => 'wait for popover on kde');
```

Comment:
Is this actually using AJAX or just an animation? If it is just an animation, `wait_for_element` would make more sense (or disabling the animation if possible).

Reply:
Yeah, after creating the PR as well I also wonder if it's ajax. I don't actually know, can you tell?

Reply:
I've just looked at the raw source code of a random overview page on o3:

```
            <tr>
                <td class="name">
                        <a data-content="<p>Testing autoyast installation with given profile and expect SLES with gnome. Please, see the profile for more details. Profile is available in distri git repo under data subdirectory. Same as autoyast_gnome but with product defined in the profile. Maintainers: riafarov, okurz</p>"
                        data-title="autoyast_gnome" data-toggle="popover" data-placement="left" data-trigger="focus" role="button" tabindex="0"><span title="autoyast_gnome">autoyast_gnome</span></a>
                </td>
```

That means there's no AJAX but the popover has an animation. I've just tested disabling it by adding the CSS rules like our `disable_bootstrap_animations` helper and it works. So it would be best to use that helper.

Reply:
using the helper now in the test module instead of waiting for ajax when there is no ajax :) I might look into disabling animations in general for our test driver in another change, not this PR.